bundles/org.openhab.binding.tesla/src/main/resources/OH-INF/thing/modely.xml

```diff
@@ -108,6 +108,7 @@
 			<channel id="shiftstate" typeId="shiftstate"/>
 			<channel id="sidemirrorheaters" typeId="sidemirrorheaters"/>
 			<channel id="smartpreconditioning" typeId="smartpreconditioning"/>
+			<channel id="steeringwheelheater" typeId="steeringwheelheater"/>
```

Comment:
Please also update the channel type description (like you did in README) and the associated entry in i18n tesla.properties file. 

Please also mention in your PR description that you added that channel for model Y,

Reply:
Please also fix the channel type to remove the readonly attribute.

Reply:
OK. Will do. Btw according to what I read this feature should be available also for Model 3 and as far as I know it was initially available for Model S/X. Should I transfer these changes to the other models (if they're not yet there)?

Reply:
Just checked now... modelX and S has the steeringwheelheater channel in the XMLs so question is now only for Model3 and Y (already done).

Reply:
https://github.com/openhab/openhab-addons/pull/13711

Reply:
You missed me! Why creating another PR for exactly the same thing? 

I had the feeling you added a second new channel corresponding to a fully different feature without saying it.

Reply:
> Btw according to what I read this feature should be available also for Model 3 and as far as I know it was initially available for Model S/X. Should I transfer these changes to the other models (if they're not yet there)?

Would make sense.

Reply:
> You lost me! Why creating another PR for exactly the same thing?
> 
> I had the feeling you added a second new channel corresponding to a fully different feature without saying it.

OK mate. I will remove this channel from the XML. I thought that you want the metadata to be pushed separately in order to appear in the release notes of the candidate with more details. Initially I wanted to implement also the wiper blades heater functionality but I couldn't find the command API call for it. I guess it makes sense to leave it for another PR.